packages/react-components/src/components/SendBox.tsx

```diff
@@ -39,6 +40,32 @@ export interface SendBoxStylesProps extends BaseCustomStyles {
   systemMessage?: IStyle;
 }
 
+/**
+ * Attributes required for SendBox to show file uploads like name, progress etc.
+ * @beta
+ */
+export interface ActiveFileUploads {
```

Comment:
this interface is named `ActiveFileUploads` but is only for one instance
```suggestion
export interface ActiveFileUpload {
```

Reply:
Good Catch

Reply:
Done